app/src/main/java/ch/epfl/sdp/contamination/DataExchangeActivity.java

```diff
@@ -41,6 +55,9 @@ public LocationService getService() {
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        sender = new ConcreteCachingDataSender(new GridFirestoreInteractor())
```

Comment:
Could get sender from LocationService?

Reply:
Yes I should make it. We should have a skype for you to explain me about this LocationService.

Reply:
We will